tests/e2e/testdata/scenarios/acquisition/sqlinstance/script.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: acquisition-sqlinstance-${uniqueId}
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: abandon
```

Comment:
nit: ABANDON sets the "deletion-policy: abandon" already. https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/378f30543ddbc50620b1d5c6fb2d02982546a389/tests/e2e/script_test.go#L296